### DISCUSSION TITLE
Retain ownership of the Lua module name "vscode"

### DISCUSSION THREAD

**Discussion by xiyaowong** [vscode-neovim/vscode-neovim#1996] [Development]
I know this is a bit of a tedious topic, but I still want to bring it up as it's been on my mind. The reason for choosing `vscode-neovim` is simply because of a conflict with an nvim color plugin. What's the big deal? The color plugin is not very useful in `vscode-neovim`, and plugin managers generally support modifying the Lua module name of plugins.

**Comment by ollien** [comment]
I guess correspondingly, what's the big deal with the existing name? I don't think it makes that much of a difference

**Reply by xiyaowong** [reply]
Because I feel like vscode-neovim is more of a runtime. For me, require'vscode' in nvim makes more sense.

**Reply by ollien** [reply]
I dunno, I think this is a small thing that has the potential to create a bit of a compatibility mess. I wouldn't lose sleep over it, personally, but I understand your preference.

**Reply by xiyaowong** [reply]
What compatibility? Just a theme plugin? Users can easily modify their configurations. Also, if we don't preserve the vscode namespace now, future modifications will become more difficult, as vscode-neovim is gaining attention and popularity.

**Reply by xiyaowong** [reply]
We need to preserve both `vscode` and `vscode-neovim`

**Reply by ollien** [reply]
> What compatibility?

Yeah, configuration.

**Reply by xiyaowong** [reply]
Among 100,000 people, perhaps only one person has this issue, and all they need to do is modify their folder name.

**Comment by theol0403** [comment]
I'm fine to do both but we don't want to break current configs

**Comment by justinmk** [comment]
100% in favor of this. The name "vscode-neovim" is very awkward to read, type, discuss, etc.

This matters because it's the primary interface for scripts, keybindings, etc. It will be typed over and over and over.
